funnel/views/proposal.py

```diff
@@ -398,16 +388,12 @@ def proposal_prev(profile, space, proposal):
     permission='move-proposal', addlperms=lastuser.permissions)
 def proposal_moveto(profile, space, proposal):
     proposal_move_form = ProposalMoveForm()
-    if not proposal_move_form.validate_on_submit():
+    if proposal_move_form.validate_on_submit():
+        target_space = proposal_move_form.target.data
+        if target_space != proposal.proposal_space:
+            proposal.current_access().move_to(target_space)
```

Comment:
Hang on, isn't there a ProposalRedirect model that must be created to manage redirects from the old URL?

Reply:
@jace ummm. I'm not sure. First time I'm hearing about it. Previous code didn't have any mention of that.

Reply:
So there is no ProposalRedirect model in the code? We need to add it then. Separate PR.

Reply:
It's mentioned only at the decorator of this view. I need to read up if that does what it's supposed to.

Reply:
It does. It's added in the old location and points to the proposal, so the browser can.be redirected. Question is, where is the code to add it?

Reply:
@jace I think here it is - https://github.com/hasgeek/funnel/pull/220/files#diff-7a3dcb6289f76819c08604a8f03c0e72L154